src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,316 @@
+---
+title: Internationalization
+description: Learn how to use i18n routing in Astro.
+i18nReady: false
+---
+
+import FileTree from '~/components/FileTree.astro'
+import Since from '~/components/Since.astro'
+
+Astro's internationalization (i18n) features allow you to adapt your project for an international audience.
+
+
+## i18n Routing (experimental) 
```

Comment:
Nit: on all the other occurrences of experimental features we added "(Experimental)" capitalized.
```suggestion
## i18n Routing (Experimental)
```